packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -542,9 +542,11 @@ class FlutterErrorDetails with Diagnosticable {
   ///     FlutterError.reportError(FlutterErrorDetails(
   ///       exception: error,
   ///       stack: stack,
-  ///       informationCollector: () sync* {
-  ///         yield ErrorDescription('This happened while climbing the space elevator.');
-  ///         yield ErrorHint('The process ID is: $pid');
+  ///       informationCollector: () {
```

Comment:
this would be a less bitter pill to swallow with the `=>` version

Reply:
Done